src/base/object_representation_sr.c

```diff
@@ -1634,6 +1652,9 @@ or_install_btids_foreign_key_ref (DB_SEQ * fk_container, OR_INDEX * index)
 
       if (set_get_element_nocopy (fk_seq, 1, &val) != NO_ERROR)
 	{
+#ifdef SERVER_MODE
+	  mmon_sub_stat_with_tracking_tag (sizeof (OR_FOREIGN_KEY));
```

Comment:
Why not subtract above two error cases, L1646 and L1638?

Reply:
Done.